# get_force_act_1.msg

string hand_name
int32[] finger_ids       
int32[] force_values     
string[] finger_names     

